### ISSUE TITLE
Manual Numbering Request

### ISSUE THREAD

**Issue by JohnHadish** [jordan2175/markdown-tools#4]
Hello @jordan2175 ,
I really enjoy your tool, I am using it for scientific writing where numbering is a must. I was frustrated that google did not have automatic numbering options for headers, so I am really happy that you implemented this.

I have a request for a new potential feature. 
I want to suppress numbering for specific headers:

Currently your tool runs like this:
![image](https://user-images.githubusercontent.com/35115540/77959840-22059300-728c-11ea-9dcc-9462f3da2736.png)

I would like my "Introduction" Section to not have a number. Example:
![image](https://user-images.githubusercontent.com/35115540/77959974-5b3e0300-728c-11ea-9f22-ac9760de6822.png)

**My ideas for implementation**

1. Have an "ignore header" option that will suppress numbering for a specific section.
2. Have a "start numbering at header `n`" setting that would skip the first `n` headings.

**Comment by jordan2175** [comment]
I have thought a lot about this in the past, I have the same problem with Appendix sections in my own documents. I do not want them to have numbers, since they are listed as Appendix A, Appendix B etc.  I have also wondered about skipping the first n number of headings, which would be far easier to implement. This would also allow you to skip the Table of Contents.  Let me think more about this.

**Comment by JohnHadish** [comment]
Sounds good

**Comment by Nixxen** [comment]
To elaborate on this, some academic writing styles require different "style" numberings/wordings on titles.
For instance IEEE formating: https://essaypro.com/blog/ieee-format#Parts
The #1 Heading starts with a roman numeral ("I. ", "II. ", "IV. " etc).
The #2 Heading starts with an uppercase letter ("A. ", "B. ", "F. " etc).
The #3 Heading starts with a number followed by parenthesis ("1) ", "2) ", "15) " etc).
The #4 Heading starts with a lowercase letter followed by perenthesis ("a) ", "b) ", "x) " etc)

A way to customize what each heading is formatted as would open up for a much wider use-case and customization of many different writing styles.

**Comment by vinodmap** [comment]
Yes please to the comment made by @Nixxen -- definitely would love to see this, same thing applies in legal writing and other areas.  I.A.1.a, etc

**Comment by zhxiaokang** [comment]
Came across a similar issue: I want to skip numbering for Abstract (1st header), but start numbering from Introduction (2nd header). Is there a way to do that?

**Comment by jcoveney279** [comment]
Having an "ignore header" option as JohnHadish suggested would be great. I use a Caption tool called Caption Maker that does this exact thing.

But here is a workaround. It's a bit of a pain but you can do this when you are ready to complete the document.

1) In the _Introduction_ Heading 1 Example, change the copy to Normal temporarily.
2) Run the Markdown tool to number everything. _Introduction_ will not be numbered but all other heading will be. _Section one_ will look like _1 Section one_
3) Make _Introduction_ Heading 1 again. It won't be numbered.
4) Update the TOC. 

![image](https://user-images.githubusercontent.com/112591247/187798434-f06aa563-dadb-4fdf-b6c3-01bb831e6e20.png)
